pipelines/managed/rh-push-to-external-registry/rh-push-to-external-registry.yaml

```diff
@@ -419,16 +419,10 @@ spec:
           value: $(tasks.collect-task-params.results.extractedValues[3])
         - name: pyxisSecret
           value: $(tasks.collect-task-params.results.extractedValues[2])
-        - name: dataPath
-          value: "$(tasks.collect-data.results.data)"
-        - name: resultsDirPath
-          value: "$(tasks.collect-data.results.resultsDir)"
         - name: ociStorage
           value: $(params.ociStorage)
         - name: sourceDataArtifact
           value: "$(tasks.create-pyxis-image.results.sourceDataArtifact)"
-        - name: dataDir
```

Comment:
https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/managed/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml#L56